### DISCUSSION TITLE
Telegram bot work

### DISCUSSION THREAD

**Discussion by ToadMr** [Gozargah/Marzban#1272] [Q&A]
Hello!
Please help me figure out the problem, why the bot in Telegram does not work correctly?
Description:
Installed Marzban.
Configured Marzban config for Telegram, as specified in the documentation.
Notifications come to Telegram (subscription created, successful/unsuccessful login), but I cannot control the bot.
There are buttons for control, I created a default template for creating a subscription, configured it according to the instructions. The point is that I can't even get information about the system from the bot.

Tell me how to overcome this?
What data do I need to provide you?

**Comment by sons-of-liberty** [comment]
/start
Provide more information, maybe a screenshot

**Reply by ToadMr** [reply]
![image](https://github.com/user-attachments/assets/b3aaa6ca-1ddb-445c-b3ca-cc04af73f23e)

When I press any button, even "System Info", the button blinks for a while, but nothing happens

What's interesting is that if I enter a command, for example
/user test
Just "test" is the login of a real user, then I get information about the user.

**Reply by ToadMr** [reply]
After reinstalling the system, the bot started working. Thank you!
